WebContent/WEB-INF/jsp/batchEdit.jsp

```diff
@@ -0,0 +1,150 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"%>
+<%@ page errorPage="/jsp/errorPage.jsp"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+
+<c:set var="tsNameLen"> ${fn:length(model.triplestore)}</c:set>  
+<c:set var="tsNameFl" scope="page">${fn:substring(model.triplestore, 0, 1)}</c:set> 
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+<jsp:include flush="true" page="/jsp/libheader.jsp" />
+<script src="//code.jquery.com/jquery-1.10.2.js"></script>
+<script src="//code.jquery.com/ui/1.11.1/jquery-ui.js"></script>
+<style>
+.i_group { width: 320px; }
+</style>
+</head>
+<body style="background-color:#fff;">
+<script type="text/javascript">
+    var letters = /^[A-Za-z]+$/;
+    function confirmEdit(){
+        var isIngest = document.getElementById("importOption").checked;
+        var preingestOption = $('input[name="preingestOption"]:checked').val();
+
+        var file = document.mainForm.dataPath.value;
+        if (file == "") {
+            alert("Please choose Excel metadata source file location.");
+            document.mainForm.dataPath.focus();
+            return false;
+        }
+
+        document.mainForm.enctype = "multipart/form-data";
+
+        var message = "Are you sure to you want to do pre-ingest validation? \n";;
+
+        if (isIngest) {
+            message = "This will update object decriptive metadata in DAMS production. \nHave all pre-ingest validations been completed? \n";
+        } else if (preingestOption == 'pre-processing')
+            message = "Are you sure to you want to review the converted RDF/XML? \n";
+
+        var exeConfirm = confirm(message);
+        if(!exeConfirm)
+            return false;
+
+        document.mainForm.action = "/damsmanager/operationHandler.do?batchEdit&progress=0&formId=mainForm&sid=" + getSid();
+        displayMessage("message", "");
+        getAssignment("mainForm");
+        displayProgressBar(0);
+        document.getElementById("import").disabled = false;
+    }
+
+    function onIngestSelectionChange(o) {
+        if ($(o).prop('name') === 'importOption') {
+            $('input[name="preingestOption"]').prop('disabled', false);
+            $('input[name="preingestOption"]').prop('checked', true);
+            $(o).prop('disabled', true);
+        } else {
+            $('input[name="importOption"]').prop('disabled', false);
+            $('input[name="importOption"]').prop('checked', true);
+            $(o).prop('disabled', true);
+        }
+    }
+
+    function reloadPage(){
+        //document.location.href="/damsmanager/batchEdit.do;
```

Comment:
Do we need this commented out still? Or, are you wanting `reloadPage()` to do anything?

Reply:
No. I'll remove it. Thanks.